webclient/static/js/leaflet.EditCoordControl.js

```diff
@@ -0,0 +1,386 @@
+                }
+
+                layer.fire('revert-edited', {layer: layer});
+            }
+        })
+    },
+    _createCoordDialog: function () {
+        this._inputcontainer = L.DomUtil.create(
+            'div',
+            'uiElement input uiHidden'
+        )
+        this._inputcontainerWGS84 = L.DomUtil.create(
+            'div',
+            '',
+            this._inputcontainer
+        )
+        this._inputcontainerJTSK = L.DomUtil.create(
+            'div',
+            '',
+            this._inputcontainer
+        )
+        let xSpan, ySpan
+
+        xSpan = L.DomUtil.create('span', '', this._inputcontainerWGS84)
+        this._inputX = this._createInput(
+            'coordInput',
+            this._inputcontainerWGS84
+        )
+        ySpan = L.DomUtil.create('span', '', this._inputcontainerWGS84)
+        this._inputY = this._createInput(
+            'coordInput',
+            this._inputcontainerWGS84
+        )
+
+        xSpanJTSK = L.DomUtil.create('span', '', this._inputcontainerJTSK)
+        this._inputXJTSK = this._createInput(
+            'coordInput',
+            this._inputcontainerJTSK
+        )
+        ySpanJTSK = L.DomUtil.create('span', '', this._inputcontainerJTSK)
+        this._inputYJTSK = this._createInput(
+            'coordInput',
+            this._inputcontainerJTSK
+        )
+        xSpan.innerHTML = ' E '
+        ySpan.innerHTML = ' N '
+        xSpanJTSK.innerHTML = ' X '
+        ySpanJTSK.innerHTML = ' Y '
+        this._buttonSubmit = L.DomUtil.create(
+            'button',
+            '',
+            this._inputcontainer
+        ) // '<button type="button" id="updateMarkerButton">Upravit</button>' +
+        this._buttonSubmit.innerHTML = this.options.modifyText
+        this._buttonSubmit.style="margin-top: 2px;"
+        this._buttonSubmit.addEventListener('click', () => {
+            this._buttonSubmitEvent();
+        })
+        L.DomEvent.on(this._inputX, 'keyup', this._handleKeypress, this)
+        L.DomEvent.on(this._inputY, 'keyup', this._handleKeypress, this)
+        L.DomEvent.on(this._inputXJTSK, 'keyup', this._handleKeypress, this)
+        L.DomEvent.on(this._inputYJTSK, 'keyup', this._handleKeypress, this)
+    },
+    _createInput: function (classname, container) {
+        let input = L.DomUtil.create('input', classname, container)
+        input.type = 'text'
+        L.DomEvent.disableClickPropagation(input)
+        return input
+    },
+    _handleKeypress: function (e) {
+        if (e.target == this._inputY || e.target == this._inputX) {
+            let x = parseFloat(this._inputX.value)
+            let y = parseFloat(this._inputY.value)
+            if (x !== undefined && y !== undefined) {
+                let coords = convertToJTSK(x, y)
+                this._inputXJTSK.value = Math.round(coords[0] * 100) / 100
+                this._inputYJTSK.value = Math.round(coords[1] * 100) / 100
+                this._epsg=4326
```

Comment:
@jhavrlant Nebylo by lepší použít konstantu?

Reply:
Zde bych to nechal pro zachování nezávislosti knihovny.